app/helpers/requests_helper.rb

```diff
@@ -100,8 +100,9 @@ def format_date(date)
     Time.zone.parse(date.to_s).strftime('%Y-%m-%d %I:%M%P')
   end
 
-  def searchworks_link(item_id, item_title)
-    link_to item_title, "#{Settings.searchworks_link}/#{item_id}", 'data-behavior': 'truncate'
+  def searchworks_link(item_id, item_title, html_options = {})
+    html_options['data-behavior'] = 'truncate'
```

Comment:
BTW, I really like the `html_options` approach.  We could have taken this approach w/ the `data-behavior` attribute initially.

Setting this attribute like this won't allow a consumer of this method to prevent truncation or passing an alternate `data-behavior` attribute.

What would you think about moving this attribute into the two places that currently call `searchworks_link`?